tools/extract_version.sh

```diff
@@ -43,7 +43,7 @@ fi
 
 if [[ "$MACHINE" == "macOS" ]]; then
     MNE_INSTALLER_NAME="MNE-Python-${MNE_INSTALLER_VERSION}-${MACHINE}_${MACOS_SUFFIX}.pkg"
-    MNE_ACTIVATE="$USER/Library/MNE-Python/bin/activate"
+    MNE_ACTIVATE="/Library/MNE-Python/bin/activate"
```

Comment:
Actually the discussion of root-only installation problems was up here: https://github.com/mne-tools/mne-installers/pull/195#discussion_r1265608601

Reply:
Please ignore my above comment for now. It's best trying to get **something** to work for now, without moving and changing permissions, since we're moving from an old version of the tools to a newer one. We may need to re-think some things.